introduction.md

```diff
@@ -1,3 +1,6 @@
 hello my name is daniel son.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?

Reply:
word